flex-template-future-cc-demo/public/appConfig.example.js

```diff
@@ -8,8 +8,9 @@ var appConfig = {
   showSupervisorDesktopView: true,
   custom_data: {
     serverless_functions_protocol: 'http',
-    serverless_functions_port: '3001',
+    serverless_functions_port: '',
```

Comment:
I reckon this line should be deleted. It isn't used in this project any more.

Reply:
agree